docs/current_docs/quickstart/agent/snippets/go/main.go

```diff
@@ -15,7 +15,7 @@ func (m *CodingAgent) GoProgram(
 	environment := dag.Env().
 		WithToyWorkspaceInput("before", workspace, "these are the tools to complete the task").
```

Comment:
```suggestion
		WithToyWorkspaceInput("before", workspace, "these are the tools to complete the assignment").
```

We use `assignment` in all the other prompts, should this also use the same term?